analytics.go

```diff
@@ -74,10 +74,7 @@ func nestedGet(key string, data interface{}) (value interface{}) {
 		switch t := value.(type) {
 		case map[string]interface{}:
 			value = t[k]
-
-		default:
-			// value doesn't exist b/c parent value isn't a map
-			panic(fmt.Sprintf("err mer gerrrd, %s doesn't exist in %s", k, key))
+			// ignore any other types/cases
```

Comment:
Tested your examples, it returns `nil` as expected. This is because it's looking for "value4" and "value3" in the "value1" blob, neither of which exist, so the `value` variable that is set and returned is `nil`.